client/state/selectors/index.js

```diff
@@ -69,6 +69,7 @@ export getReaderFollowedTags from './get-reader-followed-tags';
 export getReaderFollowForBlog from './get-reader-follow-for-blog';
 export getReaderFollowForFeed from './get-reader-follow-for-feed';
 export getReaderFollows from './get-reader-follows';
+export getReaderFollowsWithSitesAndFeeds from './get-reader-follows-with-sites-and-feeds';
```

Comment:
I'm thinking we should maybe just call this `getReaderFollows` and get rid of the old one. It can never hurt to have the site/feed attached right?